src/main/java/com/faforever/client/vault/replay/WatchButtonController.java

```diff
@@ -55,30 +51,25 @@ public void initialize() {
     delayTimeline.setCycleCount(Timeline.INDEFINITE);
 
     watchButton.setDisable(true);
+    watchButton.setOnAction(event -> {
+      getAnyPlayer(game)
+        .ifPresent(player -> replayService.runLiveReplay(game.getId(), player.getId()));
```

Comment:
Why don't we remove the plyer id from the whole request then instead of just removing the selection

Reply:
I'm hesitant to remove this information in case it ever becomes useful again for the replay server. We have to send some string to the game anyway, and doing it like this is a smaller change.

Reply:
If we wanted to, we could of course put whatever string we want here, like current user's name.

Reply:
Well this is like leaving comments in cde and i dont like that... If we ever need it again we can revert this every commit u do here or at least look into it on how to do it and copy paste

Reply:
Thats what we have git for

Reply:
In that case I can just shove current user's name there instead, sure.

Reply:
I mean what would be the use case selecting the user now that the replay sever decides which replay stream is best for us

Reply:
Removing the rest of the code reduces complexity and right now it must be really strange to read why would the client do somthing like that

Reply:
I would say lets do it clean and remove it all

Reply:
Hypothetically the replay server could match that username to whatever writer stream it has, like the legacy server did. OTOH, it would have to deal with a name not matching anything anyway, so I guess it doesn't matter either way.

Reply:
Alright, I'll do that.

Reply:
And use a dummy string f the game needs such a string